tests/test_no_ext.py

```diff
@@ -0,0 +1,16 @@
+import textract
+import os
+
+current_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Thanks for getting started with a test. Can you turn this into a proper unit test that inherits from `unittest.TestCase`?